Solutions/DNS Essentials/Analytic Rules/ExcessiveNXDOMAINDNSQueriesAnomalyBased.yaml

```diff
@@ -118,8 +118,10 @@ eventGroupingSettings:
 customDetails:
   DNSQueries: DNSQueries
   AnomalyScore: score
+  baseline: baseline
+  Total: Total
 alertDetailsOverride:
-  alertDisplayNameFormat: "[Anomaly] Excessive NXDOMAIN DNS Queries has been detected from ClientIP: '{{SrcIpAddr}}'"
-  alertDescriptionFormat: "Client source is generating excessive amount of DNS queries for non-existent domains.\n\nBaseline for 'NXDOMAIN' error count for this client: '{{baseline}}'\n\nCurrent 'NXDOMAIN' error count for this client: '{{Total}}'"
+  alertDisplayNameFormat: "[Anomaly] Excessive NXDOMAIN DNS Queries has been detected from Client IP: '{{SrcIpAddr}}'"
+  alertDescriptionFormat: "This Client source is generating excessive amount of DNS queries for non-existent domains. This can be an indication of possible C2 communications.\n\nBaseline for 'NXDOMAIN' error count for this client: '{{baseline}}'\n\nCurrent 'NXDOMAIN' error count for this client: '{{Total}}'\n\nDnsQueries requested by the client include:\n\n'{{DNSQueries}}'"
```

Comment:
Worth checking is a dynamic (DNSQueries) work in this context (as well as a custom field above).

Reply:
Thanks Ofer. It does work, I have tested mapping dynamic fields (DNSQueries, IPs) in both 'Description' as well as with 'Custom'.
![image](https://user-images.githubusercontent.com/97222872/213961898-61023232-359a-4bb5-b96b-bd910aebf271.png)

![image](https://user-images.githubusercontent.com/97222872/213962016-f5b66023-632f-4bf1-891f-81b3d4e26ab7.png)

![image](https://user-images.githubusercontent.com/97222872/213962078-4b8e3d45-9ab7-4db9-a79c-50bf03e7c736.png)